scripts/poll_spi_result

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
```

Comment:
Yes, the idea is to copy the script to the jenkins working dir, and then run it from the job. But I misunderstood the copy statement above. I'll fix that.